### PR TITLE
bill table styling

### PR DESCRIPTION
**Pull request by samuelandersoncodes**
styled bill table for responsiveness

### CODE REVIEW

**Review by douglas86 — CHANGES_REQUESTED** [review]
Is this branch updated with the main branch

File: calculated_hackers/settings.py
Comment (douglas86) [inline calculated_hackers/settings.py:1]: Lines 43, 49, 51 needs to be removed.

File: calculated_hackers/settings.py [hunk 1]
```diff
@@ -40,7 +40,15 @@
     '8000-douglas86-januaryhackat-i441hrdlu7q.ws-eu107.gitpod.io',
     'https://8000-douglas86-januaryhackat-i441hrdlu7q.ws-eu107.gitpod.io/*',
     'https://8000-douglas86-januaryhackat-i441hrdlu7q.ws-eu107.gitpod.io/accounts/login/',
+<<<<<<< HEAD
+    '8000-douglas86-januaryhackat-wlkpj1a5bcy.ws-eu107.gitpod.io'
+]
+
+CSRF_TRUSTED_ORIGINS = [
+    'https://8000-douglas86-januaryhackat-wlkpj1a5bcy.ws-eu107.gitpod.io'
+=======
```
Comment (douglas86) [inline calculated_hackers/settings.py:49]: Lines 43, 49 and 51 needs to be removed

Comment (samuelandersoncodes) [reply calculated_hackers/settings.py:49]: ok

File: templates/base.html [hunk 2]
```diff
@@ -11,104 +11,81 @@
...
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Base Template</title>
 
+    {# css styling#}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+    <link rel="stylesheet" href="{% static 'css/media_queries.css' %}">
```
Comment (douglas86) [inline templates/base.html:20]: these have already been added at line 40 to 42. These can then be deleted.

**Pull request closed without merging**